extensions/functions_comparison.yaml

```
@@ -73,4 +73,12 @@ scalar_functions:
           - value: any1
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
-
+  -
+    name: "in"
+    description: Whether a value is in a list of values.
+    impls:
+      - args:
+          - value: any1
```

I think you meant this?

```suggestion
          - value: T
```

---

The function we need is already covered by OrList expression. Removed this.